src/main/java/stsjorbsmod/patches/ExertedField.java

```diff
@@ -0,0 +1,14 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpireField;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+
+@SpirePatch(
+        clz = AbstractCard.class,
+        method = SpirePatch.CLASS
+)
+public class ExertedField {
+    @SuppressWarnings("unchecked")
+    public static SpireField<Boolean> exerted = new SpireField(() -> false);
```

Comment:
We'll probably need to add some bit of handling to make sure this state gets persisted across save/load (if we aren't already doing that for entombed, too, we probably should)

Reply:
Would it break saves if the CardData object (and corresponding json) gets Entombed and Exerted values?

Reply:
That looks good to me!